### DISCUSSION TITLE
Is there a way to customize the name and/or logo?

### DISCUSSION THREAD

**Discussion by vdrover** [mealie-recipes/mealie#4484] [Q&A]
Is there a way to customize the name "Mealie" and/or the main meailie logo on my specific self-hosted instance?

**Comment by almulder** [comment]
I too am in the need of being able to theme the app fully name, colors, and logos, including emails. And ability to hide github and docs from the app and email. Sponsor can stay but renamed to "Sponsor Developer".  And add in Help which provides the email address for the system. 

This will help my family from sending issues to them when it has nothing to do with them. and I can keep it themed to match my other services I provide. (If not willing to allow for custom name and logo, atleast the emails should match colors we setup to use) But prefer to fully theme it including emails. 

I just switched from tandoor do to a nicer interface, just lacks the themes option. Would also love to have the users be able to select from a list like tandoor

**Reply by MartinBorn** [reply]
We need this as well for our implementation. While it is possible to change it in source code, it's quite tedious to do it over and over again when updating to newer mealie version. Would be much easier if Icon and Mealie-name at the top could be configurable the same way as the colour options, outside the container.
